pkg/api/types.go

```diff
@@ -845,6 +845,9 @@ type CinderVolumeSource struct {
 	// the ReadOnly setting in VolumeMounts.
 	// +optional
 	ReadOnly bool `json:"readOnly,omitempty"`
+	// Optional: Used to configure the Cinder credentials instead of using the
+	// Cloud Provider.
+	SecretRef string `json:"secretRef"`
```

Comment:
No other optional string fields are defined as pointers in _types.go_. Could you elaborate?